pyblish/cli.py

```diff
@@ -303,4 +334,42 @@ def publish(ctx,
         click.echo(_format_time(_start, _end))
 
 
+@click.command()
+@click.argument("package", default="pyblish_qml")
+@click.pass_context
+def gui(ctx, package):
+
+    environ = os.environ.copy()
+    context = ctx.obj["context"]
+
+    with _cli_plugin(data=context.data) as plugin_path:
+        environ["PYBLISHPLUGINPATH"] = os.pathsep.join(
+            ctx.obj["plugin_paths"] + [plugin_path]
+        )
+
+        process = subprocess.Popen(
+            [sys.executable, "-m", package + ".__main__"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            env=environ,
+            bufsize=1,
+            universal_newlines=True
+        )
+        while True:
```

Comment:
Most interesting.. let's keep it the way you've got it in that case, I'll keep an eye out.